src/Service/LiveUpdates.php

```diff
@@ -76,6 +81,10 @@ public function toOrderWatchers(OrderInterface $order, $message, array $data = [
     {
         $messageName = $message instanceof NamedMessage ? $message::messageName() : $message;
 
+        if (!$this->shouldNotifyEvent($messageName)) {
```

Comment:
The live updates should still be pushed, or the UI won't update automatically. 
It's more at the email / top bar notifications level that this needs to apply.